cfgov/v1/blocks.py

```diff
@@ -117,3 +119,33 @@ class Meta:
 
     class Media:
         js = ['feedback-form.js']
+
+
+class PlaceholderFieldBlock(blocks.FieldBlock):
+    def __init__(self, *args, **kwargs):
+        super(PlaceholderFieldBlock, self).__init__(*args, **kwargs)
+        self.placeholder = kwargs.pop('placeholder', None)
+
+    def render_form(self, *args, **kwargs):
+        html = super(PlaceholderFieldBlock, self).render_form(*args, **kwargs)
+
+        if self.placeholder is not None:
+            html = self.replace_placeholder(html, self.placeholder)
+
+        return html
+
+    @staticmethod
+    def replace_placeholder(html, placeholder):
+        soup = BeautifulSoup(html, 'html.parser')
+        inputs = soup.findAll('input')
+
+        if 1 != len(inputs):
+            raise ValueError('block must contain a single input tag')
+
+        inputs[0]['placeholder'] = placeholder
+
+        return SafeText(soup)
```

Comment:
One additional note here: unfortunately, loading into BS and then re-writing as a string does have the potential to introduce minor whitespace differences between how Wagtail typically renders blocks and how these new ones would do it (no space before the closing `<input />`, for example). I didn't think this was worth the pain of implementing some alternative string replacement using regex that'd still be robust enough to e.g. quotation marks in the placeholder text.